suite-native/accounts/src/components/AccountsList/AccountsListStakingItem.tsx

```diff
@@ -20,27 +21,37 @@ export const AccountsListStakingItem = ({
     stakingCryptoBalance,
     isLast,
     ...props
-}: AccountsListStakingItemProps) => (
-    <AccountsListItemBase
-        {...props}
-        isLast={isLast}
-        showDivider={!isLast}
-        icon={<RoundedIcon name="piggyBankFilled" color="iconSubdued" />}
-        title={<Translation id="accountList.staking" />}
-        mainValue={
-            <CryptoToFiatAmountFormatter
-                value={stakingCryptoBalance}
-                symbol={account.symbol}
-                isBalance
-            />
-        }
-        secondaryValue={
-            <CryptoAmountFormatter
-                value={stakingCryptoBalance}
-                symbol={account.symbol}
-                numberOfLines={1}
-                adjustsFontSizeToFit
-            />
-        }
-    />
-);
+}: AccountsListStakingItemProps) => {
+    const { utils } = useNativeStyles();
+
+    return (
+        <AccountsListItemBase
+            {...props}
+            isLast={isLast}
+            showDivider={!isLast}
+            icon={
+                <RoundedIcon
+                    name="piggyBankFilled"
+                    color="iconSubdued"
+                    containerSize={utils.spacings.sp32}
+                />
```

Comment:
I would make the icon bit smaller
<img width="197" alt="image" src="https://github.com/user-attachments/assets/14a51084-f77e-4fa4-aec2-ac77fa1bfc5d" />
```suggestion
                <RoundedIcon
                    name="piggyBankFilled"
                    color="iconSubdued"
                    containerSize={utils.spacings.sp32}
                    iconSize="mediumLarge"
                />
```

Reply:
And the background color seems too subtle to me, but that’s a different issue.

Reply:
I'm taking my comment back, it looks better in the app with the original icon size, than on the screenshot...